Build/scripts/init_build.ps1

```diff
@@ -41,15 +99,7 @@ $CommitTime = Get-Date $CommitDateTime -Format yyMMdd.HHmm
 # TODO (doilij) refactor this into a method in a util script.
```

Comment:
Note to reviewers (and self): I've partially completed this refactoring but hadn't decided whether the block that extracts data from `$info` should also be refactored since that is done in two scripts as well.